Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/StartRide/Internal.hs

```diff
@@ -38,5 +38,5 @@ startRideTransaction driverId rideId bookingId firstPoint = do
     QBE.logRideCommencedEvent (cast driverId) bookingId rideId
     QDFS.updateStatus driverId DDFS.ON_RIDE {rideId}
     now <- getCurrentTime
-    void $ DrLoc.upsertGpsCoord driverId firstPoint now
+    void $ DrLoc.upsertGpsCoord driverId firstPoint now (Just 0)
```

Comment:
what will be it's default value?

Reply:
You can just insert `null`, because you don't have `NOT NULL `constraint